src/common/LogClient.cc

```diff
@@ -320,7 +324,10 @@ void LogClient::_send_to_mon()
   ceph_assert(messenger->get_myname().is_mon());
   ldout(cct,10) << __func__ << " log to self" << dendl;
   auto log = _get_mon_log_message();
+  #if !defined(WITH_SEASTAR) || defined(WITH_ALIEN)
```

Comment:
Not sure I understand the condition here

Reply:
crimson's Messenger is lacking of `get_loopback_connection()` implementation.